ArduCopter/GCS_Mavlink.cpp

```diff
@@ -117,7 +117,7 @@ void GCS_MAVLINK_Copter::send_position_target_local_ned()
     if (!copter.flightmode->in_guided_mode()) {
         return;
     }
-    
+
```

Comment:
Please remove these diffs (even though they're clearing out whitespace, it makes code comparison with upstream APM a PITA later)